lte/gateway/c/oai/tasks/nas/api/mme/mme_api.h

```diff
@@ -190,4 +195,42 @@ bool mme_ue_context_get_ue_sgs_neaf(
 void mme_ue_context_update_ue_sgs_neaf(
   mme_ue_s1ap_id_t mme_ue_s1ap_id,
   bool neaf);
+
+/* Compares the given two PLMNs */
+#define IS_PLMN_EQUAL(pLMN1, pLMN2, iSpLMNeQUAL)                               \
+  do {                                                                         \
+    if (                                                                       \
+      (pLMN1.mcc_digit2 == pLMN2.mcc_digit2) &&                                \
+      (pLMN1.mcc_digit1 == pLMN2.mcc_digit1) &&                                \
+      (pLMN1.mnc_digit3 == pLMN2.mnc_digit3) &&                                \
+      (pLMN1.mcc_digit3 == pLMN2.mcc_digit3) &&                                \
+      (pLMN1.mnc_digit2 == pLMN2.mnc_digit2) &&                                \
+      (pLMN1.mnc_digit1 == pLMN2.mnc_digit1)) {                                \
```

Comment:
Can we make this comparison in order as in the copy statements (i.e., mcc digits 1, 2 and3 followed by mnc digits 1, 2 and 3?